css/modal.css

```diff
@@ -0,0 +1,151 @@
+.modal {
+  display: none;
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 100vh;
+  width: 100%;
+  overflow: auto;
+  background-color: rgba(65, 63, 63, 0.6);
+}
+
+.content {
+  background-color: #fff;
+  margin: 60px auto 10px;
+  width: 88%;
+  border-radius: 8px;
+  padding: 16px;
+}
+
+.content h1 {
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+}
+
+.close {
+  float: right;
+  color: #67798e;
+  font-size: 30px;
+  transition: transform 0.2s linear;
+}
+
+.close:hover {
+  cursor: pointer;
+  transform: rotate(180deg);
+}
+
+.modal-img-container img {
+  width: 100%;
+  margin: 10px 0;
+}
+
+.modal ul {
+  padding-top: 10px;
+  width: 80%;
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  color: #6070ff;
+  margin: 10px 0;
+}
+
+.modal .below li {
+  color: #7a869a;
+}
+
+.modal hr {
+  border: 0.8px solid #ebecf0;
+  margin: 20px 0 2px;
+}
+
+.modal a {
+  text-decoration: none;
+  padding: 0;
+  margin: 0;
+}
+
+.btn-container {
+  display: flex;
+  justify-content: space-between;
+
+  /* margin: auto; */
+  padding: 0 15px;
+}
+
+.btn-container a {
+  width: 47%;
+}
+
+.modal-btn {
+  margin: 28px 0 0;
+  border: 1px solid #3965f2;
+  background-color: white;
+  padding: 8px;
+  color: #3965f2;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 15px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.modal-btn:hover {
+  background-color: #3965f2;
+  color: #fff;
+  outline: none;
+  transition: all ease-out 0.8s;
+}
+
+@media (min-width: 768px) {
+  .modal .below {
+    width: 30%;
+    margin: 0 0;
+  }
+
+  .modal .info {
+    display: grid;
+    grid-template-columns: 70% 30%;
+  }
+
+  .modal .side {
+    margin: 0 20px;
+    padding: 0;
+  }
+
+  .modal .skills {
+    width: 75%;
+  }
+
+  .modal-wrapper {
+    margin: 24px auto;
+    width: 88%;
+    padding: 40px 0;
+  }
```

Comment:
Great design @AbdimulhinYussuf3675 🤝

- [ ] Kindly fix the responsiveness correctly. Your buttons seem to overlap as soon as you hit the 800px screen size.
![responsive](https://user-images.githubusercontent.com/63917725/196876340-3f22715f-a523-4660-90c1-ad988a7abd02.PNG)